test/models/environment_test.rb

```diff
@@ -5,6 +5,10 @@ class EnvironmentTest < ActiveSupport::TestCase
     Settings.puppet_db.enabled = false
   end
 
+  teardown do
+    Settings.puppet_db.enabled = false
```

Comment:
if `puppet_db` is enabled, then needs a PuppetDB running or stub the PuppetDB client requests

Reply:
Um, sure. This is what happens in one of the tests.

This change is just to make sure the setting is turned off again so no other tests are affected.

Reply:
Unit tests will run on system without PuppetDB. So we need a stub like providing fix test data.